packages/plugin/src/rules/graphql-js-validation.ts

```diff
@@ -44,11 +45,16 @@ export function validateDoc(
   }
 }
 
+const isGraphQLImportFile = rawSDL => {
```

Comment:
maybe it's better to export and import this function? https://github.com/ardatan/graphql-tools/blob/master/packages/loaders/graphql-file/src/index.ts#L35

Reply:
Yeah, I agree, but it shouldn't be a blocker for this PR. @ardatan can you please export this function and release a new version?